fix dependency error

markusstraub opened the pull request:
sorry.. changing `op` to `predicate` requires at least geopandas 0.10
see https://geopandas.org/en/stable/docs/changelog.html

fredshone commented:
I guess this specific to one of your processes or platform, but happy to bump versions

markusstraub commented:
The problem was not triggered by the tests as far as I remember but only happened when trying to do some facility sampling. But thanks for merging anyways :)

You might also be interested in https://github.com/markusstraub/pam/tree/py311 .. I updated the dependencies to work with python 3.10 and 3.11. I fixed the dependencies again because pandas and other libraries evolve so quickly.

fredshone commented:
ah thanks - might steal this - we are in the process of getting everything to 3.11
